documentation/modules/ROOT/pages/04-scaling.adoc

```diff
@@ -21,8 +21,7 @@ include::partial$build-containers.adoc[tag=greeter]
 [#scaling-modify-configmap]
 == Setting Configmap Defaults
 
-Knative v0.4.0 changed how the default configuration map values are set and provided.
```

Comment:
No need to state this changed happened in 0.4.0 IMO as we're already at 0.5.0

Reply:
:+1: